### PR TITLE
Updated Searchbar Input from MUI to Tailwind CSS

### PR DESCRIPTION
**Pull request by Nishitbaria**
## Related Issue

Closes: #643 

## Description of Changes

This pull request addresses the issue of longer build times caused by using Material-UI (MUI) in the Searchbar component. To improve our application's performance and optimize the build process, I have updated the Searchbar Input component from MUI to Tailwind CSS.

Changes Made:
- Replaced the MUI Searchbar Input component with the equivalent component using Tailwind CSS.
- Updated necessary CSS styles to align with our application's design requirements.

Benefits of the Update:
- Reduced build times: By using Tailwind CSS, we can take advantage of its lightweight nature, resulting in faster build times during development.
- Improved application performance: Tailwind CSS allows us to reduce the overall CSS footprint, leading to quicker load times for our users.
- Enhanced customizability: With Tailwind CSS, we have greater flexibility in customizing styles to match our application's design and branding.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots
image

### CODE REVIEW

**Comment by anmode** [comment]
Fix prettier please and it is good to merge

File: components/hackthons/components/Searchbar.js [hunk 1]
```diff
@@ -1,151 +1,150 @@
...
-    </form>
-  );
-}
-
-export default SearchBar;
+import React, { useState } from "react";
+
+// import styles from '../styles/Searchbar.module.css'; // Import the CSS file
+
+import Chip from "@mui/material/Chip";
```
Comment (anmode) [inline components/hackthons/components/Searchbar.js:5]: you are using MUI :)

**Review by anmode — CHANGES_REQUESTED** [review]

File: components/hackthons/components/Searchbar.js (hunk 1 above)
Comment (Nishitbaria) [reply components/hackthons/components/Searchbar.js:5]: In thi pull request ,i have fixed  search input fieldas mentioned ,later i will fix the chips and make pr for it

**Comment by Nishitbaria** [comment]
@anmode ,i have updated code through prettier extension and also checked online still it is failing

File: components/hackthons/components/Searchbar.js (hunk 1 above)
Comment (anmode) [reply components/hackthons/components/Searchbar.js:5]: Chip is part of search bar, please you have to fix ik this pR only.

**Comment by anmode** [comment]
> @anmode ,i have updated code through prettier extension and also checked online still it is failing

No worries. I'll see prettier.

**Comment by Nishitbaria** [comment]
@anmode ,chips fixed now please review and merge my pull request

**Comment by anmode** [comment]
Screenshot 2023-08-02 at 10 35 12 AM
Not responsive!

**Review by anmode — APPROVED** [review]
Just a last fix! if you see in the production it look really cool.
1. The color please use the our theme color in tag and in searchbar 
2. I see on tag there is nice hover effect on click ( see production: www.grabtern.com)

1st point is main , 2nd point is not mandatory to be cover....
btw nice!! PR. Hope to get more to completely remove MUI and to use tailwind

**Comment by Nishitbaria** [comment]
@anmode ,please review and merge my pull request

**Review by anmode — APPROVED** [review]
Cool! Hope to see more follow-ups pR

**Pull request merged**